### ISSUE TITLE
ATM's withdrawls only, Banks to deposit cash

### ISSUE THREAD

**Issue by BIRLZ95** [New-Chance-RP/Feedback#732]
<h2>Feature you are requesting.</h2>

At the moment you can put money into wall ATM's and withdrawn money also. 

I'm requesting only be able to withdraw money from ATM's maybe 5K max? and must visit a bank to deposit your cash or withdraw a higher ammount.

<h2>What are the benefits to your feature?</h2>

Get people visiting banks more
Realism 

<h2>Anything to support your feature request?</h2>

Community requested by Lugi

**Comment by Mubbb** [comment]
Makes sense to be honest. Always found it odd that you can deposit at ATMs but figured it was just for ease.

**Comment by Zopfco** [comment]
I assumed it was at least partially to make it less attractive to post up in a bank with a shotgun and rob everyone that comes in?  If we're talking about realism, people don't do this because banks also have armed security and functional cameras (not to mention an overwhelming 24/7 police response.)

**Comment by TaxEhh** [comment]
I like the idea of a max limit for withdrawals from an ATM.
Anything higher then that max amount, you must go into a bank like Zopfco said for security and just more realism of it being a large amount of money.

You should be able to deposit as much money as you want however into ATMs.  Not sure if this is possible but if you wanted to make it more realistic maybe make depositing to the machine go slower if it's a LARGE amount of cash due to the number of bills/coins you would be depositing since you arent just stuffing it all in on one go.

**Comment by Mubbb** [comment]
> I assumed it was at least partially to make it less attractive to post up in a bank with a shotgun and rob everyone that comes in? If we're talking about realism, people don't do this because banks also have armed security and functional cameras (not to mention an overwhelming 24/7 police response.)

Yeah but people like that are the same people that have shootouts at burger shot as if the largest PD in the city isnt at the side of them. They should be dealt with via staff if they're gonna sit in a bank with a shotgun and wait for people to walk in with cash.

Depositing money via ATM makes no sense IMO, maybe multiple banks would help, or even dropbox locations like the one attached to the bank in the north of the city.

**Comment by Ic3BoxTheSuspect** [comment]
Although in real life you can deposit cash into _some ATMs,_ you can't deposit cash into **every** ATM you see on the street. 

And since it would probably be an unecessary pain in the ass to sort out/script which ATMs would allow cash deposits (example those outside banks), I agree that money should only be deposited at banks.

**Comment by Senlarus** [comment]
I worked in the banking industry for 6 years in the US. Every bank ATM is capable of accepting cash deposits. The kind at 24/7 are not because those are not from banks. But it’s not worth scripting out the different ATMS to behave in different ways. Same with adding a cash limit, too much code required to support that for not enough benefit.

so we’re left with a yes or no situation.

**Comment by Senlarus** [comment]
I can set a max per transaction though which would make it a pain in the ass to do large amounts...

**Comment by Mubbb** [comment]
Fair enough. I've personally never heard of or seen an ATM that can accept deposits so I figured it was ingame arms basically sharing the same script as the banks for ease. But yeah that's fair, probably not worth messing with
